### PR TITLE
Code review request

### PR DESCRIPTION
**Pull request by ebukaume**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,844 @@
...
+      href="./assets/banner/favicon.png"
+      type="image/x-icon"
+    />
+    <title>
+      Space Ripples Reveal Big Bang’s Smoking Gun - The New York Times
+    </title>
+  </head>
+  <body>
+    <nav>
+      <div id="menu"><i class="fas fa-bars"></i></div>
```
Comment (AdnanAfsari) [inline index.html]: use class instead

File: index.html [hunk 2]
```diff
@@ -0,0 +1,844 @@
...
+            <button>READ 615 COMMENTS</button>
+            <span class="social-media fab fa-facebook-f"></span>
+            <span class="social-media fab fa-twitter"></span>
+            <span class="social-media far fa-envelope"></span>
+            <span class="social-media fas fa-share"></span>
+          </div>
+        </section>
+      </article>
+      <div class="editors-pick-vertial">
+        <div id="top">
```
Comment (AdnanAfsari) [inline index.html]: use class instead

File: index.html [hunk 3]
```diff
@@ -0,0 +1,844 @@
...
+              <li>
+                <img
+                  src="./assets/images/sub-cooking.png"
+                  alt="Cooking subscription"
+                />
+                Cooking
+              </li>
+            </ul>
+          </div>
+          <div id="sub-mid">
```
Comment (AdnanAfsari) [inline index.html]: use class instead

**Comment by ebukaume** [comment]
Hello @AdnanAfsari 

Thanks for your review. The reason we used id is because we can to uniquely identify those elements. If we change those to classes, then it means we are creating a class to be used only in one place.

Our is that not reason enough to use id?

**Comment by AdnanAfsari** [comment]
Hello @ebukaume @Forison 
Thanks very much for your patient and hard work.

- If you read about  best practices of css, you will find that they suggest to use class instead of id whenever possible, only use id where you have to use :).so its up to you actually where to use what following best practices :)

- The Theory of Inflation section is still broken. Please fix that. everything looks great except this section.
Screenshot 2019-04-09 at 7 50 03 AM

- Please submit for another review after fixing that section.

File: index.html (hunk 3 above)
Comment (Forison) [reply index.html]: hi @AdnanAfsari ,please can you tell us which browser you used ?

**Comment by ebukaume** [comment]
Thanks @AdnanAfsari 

We were able to see the issue on Opera. We have fixed it. It now works well across all the browsers we have tested on.

**Comment by AdnanAfsari** [comment]
Hi guys, everything looks perfect now 👍 
you have completed the project successfully, please go ahead and merge the branch.

Happy coding :)

**Comment by ebukaume** [comment]
Thank you @AdnanAfsari. We will do that right away.

**Pull request merged**